tests/functional/codegen/features/iteration/test_for_range.py

```diff
@@ -89,7 +184,7 @@ def test_offset_repeater_2(get_contract_with_gas_estimation, typ):
 @external
 def sum(frm: {typ}, to: {typ}) -> {typ}:
     out: {typ} = 0
-    for i in range(frm, frm + 101):
+    for i in range(frm, frm + 101, bound=101):
```

Comment:
just as a note, a lot of these can be refactored now. like this could be rewritten as
```vyper
for i in range(frm, to, bound=101):
    ...
```